test/integration/suites-windows/windows-service/00-setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+pwd
```

Comment:
why pwd is required?

Reply:
Not sure if it is actually required, I took it from another test sample (windows workload attestor). I'm going to check if it is really necessary.